book/src/getting-started/4-cargo-executable.md

```diff
@@ -27,6 +27,8 @@ Add the dependencies to the `Cargo.toml` file.
 We'll need `cxx`, `cxx-qt`, `cxx-qt-lib` and `cxx-qt-build`:
 
 ```toml,ignore
+[package]
+name = "qml_minimal"
```

Comment:
could use the package info from the examples/qml_minimal toml ? to try and keep as much as possible from real code?

Reply:
Done